src/main/java/net/amygdalum/tanteemmas/server/Server.java

```diff
@@ -84,8 +85,10 @@ public void order(RoutingContext context) {
 		}
 		PriceCalculator prices = new PriceCalculator(date, daytime, weather);
 		String name = context.request().getParam("product");
-		Map<String, Object> product = products.getProduct(name);
-		prices.order(product);
+		Map<String, Object> product =  new HashMap<>( products.getProduct(name));
```

Comment:
Looks like there're are two whitespaces in front of `new HashMap...`? And another one before `products...`?